analyzer/metadata-plugin/src/main/java/eu/fasten/analyzer/metadataplugin/MetadataDBExtension.java

```diff
@@ -165,7 +173,15 @@ public void consume(String record) {
                         setOutputPath(callgraph);
                     }
                 });
+            } catch (DataAccessException e) {
+                // The error codes starting with 57P0 are related to the DB connection issues.
+                // See https://www.postgresql.org/docs/current/errcodes-appendix.html
+                if (e.sqlState().contains("57P0")) {
```

Comment:
See previous discussion

Reply:
Addressed in [b9dbc27](https://github.com/fasten-project/fasten/pull/466/commits/b9dbc27c6b91895e7a28b43d985207bf883e22b7).